### ISSUE TITLE
Error: Cannot find module '@sls-next/next-aws-cloudfront'

### ISSUE THREAD

**Issue by tyler-ground** [serverless-nextjs/serverless-next.js#478]
**Describe the bug**
When deploying serverless I am suddenly getting the above error

**To Reproduce**
Deploy project

**Expected behavior**
Successful deploy

**Screenshots**

**Desktop (please complete the following information):**

- OS: OSX
- Version: 1.15.0-alpha.2

**Comment by lone-cloud** [comment]
Have you tried using the latest `component: "@sls-next/serverless-component@1.15.0-alpha.4"`?

**Comment by tyler-ground** [comment]
small tweak, I had an up arrow in package.json so I think it's actually running "1.15.0-alpha.2"

**Comment by tyler-ground** [comment]
@lone-cloud oh, the latest on npm is alpha.2

**Comment by lone-cloud** [comment]
We very recently switched package names. The latest is here: https://www.npmjs.com/package/@sls-next/serverless-component

**Comment by tyler-ground** [comment]
ohhhhhhh,  THANKS!

**Comment by tyler-ground** [comment]
Can we add a pointer to the NPM page to the new package?

**Comment by lone-cloud** [comment]
A question for @danielcondemarin

**Comment by tyler-ground** [comment]
Downgrading to 1.14.0 did fix this, I assume the latest alpha will fix it as well :)

**Comment by tyler-ground** [comment]
spoke too soon, downgrading did not fix it, going to try alpha 4

**Comment by tyler-ground** [comment]
still failing, going to see where the bug is.

**Comment by danielcondemarin** [comment]
> Can we add a pointer on the NPM page to the new package?

I am planning to deprecate it once the latest alpha is graduated to stable.

**Comment by tyler-ground** [comment]
Okay, I made an error when I tried the new package (needed to update the name in serverless.yml). Deploying again

**Comment by tyler-ground** [comment]
Deployment Succeeded!

TLDR: If you are hitting this error switch to the new package path (and also update serverless.yml)
